docs/Using-Docker.md

```diff
@@ -1,6 +1,7 @@
 # Using Docker For ML-Agents (Experimental)
 
-We currently offer an experimental solution for Windows and Mac users who would like to do training or inference using Docker. This option may be appealing to those who would like to avoid installing Python and TensorFlow themselves. The current setup forces both TensorFlow and Unity to _only_ rely on the CPU for computations. Consequently, our Docker support is limited to environments whose agents **do not** use camera-based visual observations. For example, the [GridWorld](Learning-Environment-Examples.md#gridworld) environment is **not** supported.
+We currently offer a solution for Windows and Mac users who would like to do training or inference using Docker. This option may be appealing to those who would like to avoid installing Python and TensorFlow themselves. The current setup forces both TensorFlow and Unity to _only_ rely on the CPU for computations. Consequently, our Docker simulation [uses the CPU](https://en.wikipedia.org/wiki/Xvfb) to do visual rendering. This means that environments which involve agents using camera-based visual observations might be slower.
```

Comment:
- Linking to https://en.wikipedia.org/wiki/Xvfb in "uses the CPU" was a bit counter-intuitive for me.
- The very end of this sentence "might be slower", presumably you mean training and inference speeds. I would clarify that. (Optional: is there anything we can say regarding how much slower we expect it to run?)

Reply:
> Linking to https://en.wikipedia.org/wiki/Xvfb in "uses the CPU" was a bit counter-intuitive for me.
`Xvfb` is a virtualized XServer which does not use the GPU, would you prefer if the text hyperlinked said `does not use the GPU` ? 

> The very end of this sentence "might be slower", presumably you mean training and inference speeds. I would clarify that. (Optional: is there anything we can say regarding how much slower we expect it to run?)

I mean training. Although you could very well use the container for inference as well, it seems more designed for a training use case. I will clarify there. 

We are not significantly slower for the environments we support currently. Having said that, we are using the CPU to do computation which is typically done in a GPU. So when we do get to more complex and rich rendering scenes, I expect performance to drop. That phrase was mostly added to ward off potential issues popping up on Github in the mean time.

Reply:
I think if we're going to link to Xvfb then we should probably introduce it here too.